books/RayTracingInOneWeekend.html

```diff
@@ -616,9 +677,9 @@
 
 <div class='together'>
 The `ray_color(ray)` function linearly blends white and blue depending on the height of the $y$
```

Comment:
Don't have the `ray_color` function in the above code block. There's already a lot going on in that.

We also completely dropped the motivation for adding a `ray_color` since we just got inundated with camera stuff. Move line 514-515 to here, and then make a code block for `ray_color`